src/components/remotecontrol/remotecontrol.scss

```
@@ -327,8 +325,13 @@
     }
 
     .nowPlayingPageImageContainer {
+        flex-grow: 1;
+        flex-shrink: 1;
         width: 100%;
-        margin: auto auto 0.5em;
+        min-height: 0;
+        margin: 0 auto 0.5em;
+        display: flex;
+        align-items: center;
     }
```

For Chrome 79 - it stretches the image when height is small.
Also Safari 10 starts shrinking the image, but doesn't respect the aspect ratio.
```suggestion
        flex-direction: column;
        justify-content: center;
        align-items: center;
    }
```

---

I'm not sure `justify-content` is actually doing anything here, but I added it in case it has some effect on one of the problematic browsers. :smile:

---

Since we are changing the main axis in this way, we should use `justify-content` for vertical alignment.

There seems to be a downside: the poster is horizontally out of its container because it has a border (overall size > 100%), and `align-items` doesn't help.
What about `box-sizing: border-box;` on the image?

---

It was hard for me to notice the misalignment in my testing. I did remove the border because it seemed out of place and added the same border radius we use on item cards.